components/script/dom/identityhub.rs

```diff
@@ -64,8 +64,8 @@ pub struct Identities {
     dummy_hub: IdentityHub,
 }
 
-impl Identities {
-    pub fn new() -> Self {
+impl Default for Identities {
+    fn default() -> Self {
```

Comment:
You also need to update all references to `Identities::new()` to be `Identities::default()` if you change the name of the method.